internal/grid/muxclient.go

```diff
@@ -347,8 +347,14 @@ func (m *muxClient) handleOneWayStream(respHandler chan<- Response, respServer <
 				m.addErrorNonBlockingClose(respHandler, ErrDisconnected)
 				return
 			}
-			// Send new ping.
-			gridLogIf(m.ctx, m.send(message{Op: OpPing, MuxID: m.MuxID}))
+			select {
```

Comment:
You are not removing the issue, just making it a tiny bit less likely. I will send a PR.

(the mux can still be closed between checking the context and sending the message)

Reply:
Okay